container-images/cann/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+export LD_LIBRARY_PATH=/usr/lib:${LD_LIBRARY_PATH}
+
+cann_in_sys_path=/usr/local/Ascend/ascend-toolkit
```

Comment:
Since we control what is put in the containerfile, why do we need this entrypontcode outside of the Containerfile?  The environemtn variable and entrypoint should be there, or am I missing something?